tests/modules/images/webp-uploads/webp-uploads-test.php

```diff
@@ -3,9 +3,12 @@
  * Tests for webp-uploads module.
  *
  * @package performance-lab
+ * @group webp-uploads
  */
 
 class WebP_Uploads_Tests extends WP_UnitTestCase {
+	function return_webp_array() {
+		return array( 'image/webp' ); }
```

Comment:
The syntax here looks off? Can you move the closing brace below?

Reply:
Ah I see you changed it now, nevermind.

Reply:
oof phpcbf did that, i was shooting for a single line function which phpcs didn't like; fixed.